Scrapers/Huone.py

```diff
@@ -0,0 +1,96 @@
+from bs4 import BeautifulSoup
+from requests import get
+import VenueClass
+import json
+from DataFormatting.FormatTime import FormatTime
+from DataFormatting.FormatPrice import FormatPrice
+from DataFormatting.FormatText import FormatTitle
+
+allVenues = []  # Array to store all Venues of type VenueClass
+
+def extractVenue(tag, venue):
+
+    # extract link into the venue
+    venue_link = venue.find_all('a')[0]['href']
+    response = get(venue_link)
+    indiv_venue = BeautifulSoup(response.text, 'html.parser')
+
+    # extract title
+    title_html = indiv_venue.find('div', class_='space-title').find('h1')
+    title = ''
+    if title_html:
+        title = FormatTitle(title_html.text.strip())
```

Comment:
Since the venues listed in this website is in the same location, the title here are just the room names. Perhaps you could append a HUONE in front of the title? 

Also, correct me if i am wrong, the title_html.text.strip() seems to be in the correct format already (First character of a word is capitalised and the rest lower caps) hence i think we can omit the FormatTitle to make the scraper more efficient. Tbh, the formatting code is not very well written by me. It is quite complex so i think we should only use it when needed.

line 22
title = "HUONE "+ title_html.text.strip()

Reply:
Yupp the venues are all like that: Cottage Room - the first letter of each word is capitalised. have removed the FormatTitle!